plugin-dev/Source/Sentry/Private/Android/SentrySubsystemAndroid.cpp

```diff
@@ -169,6 +169,32 @@ USentryId* SentrySubsystemAndroid::CaptureEventWithScope(USentryEvent* event, co
 	return SentryConvertorsAndroid::SentryIdToUnreal(*id);
 }
 
+USentryId* SentrySubsystemAndroid::CaptureException(const FString& type, const FString& message, int32 framesToSkip)
+{
+	// add marker tags specific for Unreal assertions
+	SetTag(TEXT("sentry_unreal_exception"), TEXT("assert"));
+	SetTag(TEXT("sentry_unreal_exception_skip_frames"), FString::Printf(TEXT("%d"), framesToSkip));
+	SetTag(TEXT("sentry_unreal_exception_type"), type);
+	SetTag(TEXT("sentry_unreal_exception_message"), message);
```

Comment:
Can these be put in `contexts` instead?

Reply:
The problem with using context values here is that these are ignored by `sentry-native` when it captures a native crash on Android and thus there will be no way to identify whether the event is an assertion or not upon the next application launch.